docs/source/new-docs/getting-started/system-requirements/index.rst

```diff
@@ -8,30 +8,84 @@
-     - This is a Reference page - please consult `this guide <https://docs.divio.com/documentation-system/reference/>`_
+We officially support RedHat and Debian flavors of Linux. We aim to also support MacOS, and Windows on
+both x86 and ARM processors. If you encounter an issue on Mac or Windows,
+please submit an issue and while we cannot guarantee anything, we will do our best to address it.
 
- - A PR from your feature branch to ``main`` 😊
++------------------+---------------------------------------------------------+
+| Operating System | Minimum Version                                         |
++==================+=========================================================+
+| Linux            | RedHat 8, AlmaLinux 8, RockyLinux 8 equivalent or later |
++------------------+---------------------------------------------------------+
+|                  | Debian 10 equivalent or later                           |
++------------------+---------------------------------------------------------+
+| MacOS            | 12 or later *(not officially supported)*                |
++------------------+---------------------------------------------------------+
+| Windows          | 10 or later *(not officially supported)*                |
++------------------+---------------------------------------------------------+
+
+At the moment, the installation instructions for GeoIPS (which you can find [HERE])
+work for these OS/processor archetype combos:
+
++---------------------------+--------------------+--------------------+--------------------+------------------------+
+|                           | Conda              | Expert Install     | Docker (native)    | Docker (x86 emulation) |
++===========================+====================+====================+====================+========================+
+| Debian (x86)              | ✅                 | ✅                 | ✅                 | N/A                    |
++---------------------------+--------------------+--------------------+--------------------+------------------------+
+| Redhat (x86)              | ✅                 | ❓                 | ❓                 | N/A                    |
++---------------------------+--------------------+--------------------+--------------------+------------------------+
+| Mac (x86)                 | ❓                 | ❓                 | ❓                 | N/A                    |
++---------------------------+--------------------+--------------------+--------------------+------------------------+
+| Mac (arm)                 | ❌                 | ❓                 | ❌                 | ✅                     |
++---------------------------+--------------------+--------------------+--------------------+------------------------+
+| Windows with WSL2 (x86)   | ❓                 | ❓                 | ❓                 | N/A                    |
++---------------------------+--------------------+--------------------+--------------------+------------------------+
```

Comment:
What are the plans for keeping this matrix up to date?  Just update every time installation instructions are updated?  Is there a regular testing process for each of the green check marks ensuring it continues to be functional?

Reply:
That's a really good question. We don't really have a process for testing installation at the moment, so I'm not sure. Some of this could be automated via the CLI (eg. nightly builds or something like that). Maybe something to discuss after the workshop?

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/issues/719

I created an issue, will resolve and approve this.